exporter/configCli.php

```diff
@@ -0,0 +1,10 @@
+<?php
+          define('DATABASE_HOST', 'localhost');
```

Comment:
@eduardBoikov one big difference I notice in this script vs the web script, we wanted the user to not have to setup a config file.

In the web script if they run it from the correct folder, we detect and copy these configuration values from the already existing config file.

Can you make it so this script does the same thing?

If you can't find the existing config file, maybe it just asks the values with stdin instead of a config file?